denops_std/helper/load_test.ts

```diff
@@ -14,13 +14,54 @@ test({
   mode: "any",
   name: "load() load remote Vim script file",
   fn: async (denops) => {
+    const loadScriptUrl =
+      "https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test.vim";
```

Comment:
Please define the base URL in file-level and use it like

```typescript
const loadScriptUrl = new URL("load_test.vim", loadScriptUrlBase);
await load(denops, loadScriptUrl);
// ...
const loadScriptUrl = new URL("load_test_not_exists.vim", loadScriptUrlBase);
await load(denops, loadScriptUrl);
```

Reply:
Fixed it with 6d7fdec.